clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -26,24 +21,37 @@ import shapeless.CNil
 sealed trait ClioCommand
 
 case object GetWgsUbamSchema extends ClioCommand
+case object GetGvcfSchema extends ClioCommand
```

Comment:
It also occurs to me that, in order to find all the related names easily, it would be better to put the unique part of the name at the end or the beginning. E.g. `GetSchemaGvcf` or GvcfGetSchema`. The same rule would apply to `Input`, `Metadata`, etc. I think having the unique part first might be better since they would all sort together.